application/event_processor/changes.py

```diff
@@ -32,20 +33,20 @@ def get_changes(dos_service: DoSService, nhs_entity: NHSEntity) -> Dict[str, str
     return changes
 
 
-def update_changes(changes: dict, change_key: str, dos_value: str, nhs_uk_value: str) -> None:
+def update_changes(changes: dict, change_key: str, dos_value: str, nhs_uk_value: Any) -> None:
     """Adds field to the change request if the field is not equal
     Args:
         changes (dict): Change Request changes
         change_key (str): Key to add to the change request
         dos_value (str): Field from the DoS database for comparision
-        nhs_uk_value (str): NHS UK Entity value for comparision
+        nhs_uk_value (str|None): NHS UK Entity value for comparision
 
     Returns:
         dict: Change Request changes
     """
     if str(dos_value) != str(nhs_uk_value):
         logger.debug(f"{change_key} is not equal, {dos_value=} != {nhs_uk_value=}")
-        changes[change_key] = nhs_uk_value
+        changes[change_key] = nhs_uk_value if not is_val_none_or_empty(nhs_uk_value) else ""
```

Comment:
this will still send a change though with empty string which is what it was doing before, then DoS will through it out as its None / Null in the DB